src/styles/index.scss

```diff
@@ -1,3 +1,122 @@
+$main: #060b35;
+$secondary: #616070;
+
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
 }
+
+.card {
+  flex-direction: column;
+  width: 200px;
+  border: solid 1px #f3f3f3;
+  display: flex;
+  border-radius: 5px;
+  font-family: "Roboto", sans-serif;
+  padding: 32px 16px 16px;
+}
+
+.card__picture {
+  width: 160px;
+  height: 134px;
+  margin: 0 auto 40px;
+}
+
+.card__name {
+  margin: 0 0 4px;
+  color: $main;
+  width: 166px;
```

Comment:
Setting widths and heights as fixed values is generally a bad idea most of the times. Such elements won't be responsive. It's better to set their widths to 100%.

Reply:
Heights can be managed by just line-height for texts and/or margins/paddings